src/field_theory/is_alg_closed/classification.lean

```diff
@@ -177,7 +177,7 @@ begin
 end
 
 private lemma ring_equiv_of_cardinal_eq_of_char_p (p : ℕ) [fact p.prime]
```

Comment:
It is very strange that this is a lemma

Reply:
I made it a def, I wonder why the linter didn't complain.

Reply:
I imagine because it is `private`.